legacy/public.styl

```diff
@@ -10205,7 +10226,20 @@ section.contact-departments .contact-wrapper:hover .contact-metadata,
     background-color: white;
     border-color: #e0e0e0;
     color: #288be4
+
+    &:not(.active):hover{
+        // du ticket #973
+        background-color: #288be4;
```

Comment:
Looks like we haven't been doing this in the legacy code, but it might be a good idea to put colors in variables so we can can them in one place. Whatever you think is best works for me.